Common.tex

```diff
@@ -198,6 +198,12 @@ \section{Group Functions}\label{sec:Group}
 Groups can only contain objects from a single \texttt{PWR_cntxt}.
 Group operations that involve multiple groups must be performed with groups from the same context.
 
+Some groups may be created automatically by an implementation for the convenience of the user. Examples of such groups include
+a group of nodes assigned to a job or a group of objects being cooled by a CDU object. An implementation may choose to provide
+a group (or groups) of redundancy for CDU overlap where failures in a single CDU can be compensated for by other existing CDUs.
```

Comment:
```suggestion
a group (or groups) to represent which CDUs overlap so that failures in a single CDU can be compensated for by other existing CDUs.
```